src/codeu/chat/database/Database.java

```diff
@@ -0,0 +1,91 @@
+package codeu.chat.database;
+
+import codeu.chat.common.Conversation;
+import codeu.chat.common.Message;
+import codeu.chat.common.User;
+import codeu.chat.util.Uuid;
+import com.mongodb.MongoClientURI;
+import org.bson.Document;
+
+import com.mongodb.MongoClient;
+import com.mongodb.client.MongoCollection;
+import com.mongodb.client.MongoDatabase;
+
+public class Database {
```

Comment:
Let's use a more descriptive noun. Is this a ChatAppDatabaseConnection, or what is it?